src/properties.js

```diff
@@ -12,115 +122,30 @@ const enrichPropertyDefinitions = (propertyDefinitions) => {
-              timeZoneName: "short"
-            }) + suffix;
+    formatSummaryValues: (key, values) => {
+      if (Array.isArray(values)) {
+        let list = values.sort().map(v => formatValue(key, v))
+        if (list.length > 1) {
+          return "<ul><li>" + list.join("</li><li>") + "</li></ul>";
         }
-
-        if (propertyMap[key].type === "label:property") {
-          if (value == null) {
-            return undefined;
-          }
-
-          return value.map(x => `<code>${x}</code>`).join(", ");
-        }
-
-        if (propertyMap[key].type === "label:classes") {
-          if (Array.isArray(value)) {
-            return value
-              .map(o =>
-                Object.entries(o)
-                  .map(([k, v]) => {
-                    if (k === "name") {
-                      if (v === "raster") {
-                        return undefined;
-                      }
-
-                      return `<code><b>${v}</b></code>:`;
-                    }
-
-                    if (Array.isArray(v)) {
-                      return v.map(x => `<code>${x}</code>`).join(", ");
-                    }
-
-                    return v;
-                  })
-                  .join(" ")
-              )
-              .join("<br>\n");
-          }
-
-          return Object.entries(value)
-            .map(([k, v]) => {
-              if (k === "name") {
-                if (v === "raster") {
-                  return undefined;
-                }
-
-                return `<code><b>${v}</b></code>:`;
-              }
-
-              if (Array.isArray(v)) {
-                return v.map(x => `<code>${x}</code>`).join(", ");
-              }
-
-              return v;
-            })
-            .join(" ");
-        }
-
-        if (propertyMap[key].type === "label:overviews") {
-          return value
-            .map(v => {
-              const prop = v.property_key;
-
-              if (v.counts != null) {
-                return `<code><b>${prop}</b></code>: ${v.counts
-                  .map(c => `<code>${c.name}</code> (${c.count})`)
-                  .join(", ")}`;
-              }
-
-              if (v.statistics != null) {
-                return `<code><b>${prop}</b></code>: ${v.statistics
-                  .map(c => `<code>${c.name}</code> (${c.count})`)
-                  .join(", ")}`;
-              }
-
-              return "";
-            })
-            .join("<br>\n");
+        else {
+          return list[0];
         }
       }
-
-      if (key === "eo:epsg") {
-        return `<a href="http://epsg.io/${value}">${value}</a>`;
-      }
-
-      if (Array.isArray(value)) {
-        return value.map(v => {
-          if (typeof (v) === "object") {
-            return JSON.stringify(v);
-          }
-          return v;
-        });
-      }
-
-      if (typeof value === "object") {
-        return JSON.stringify(value);
+      else if (values && typeof values === 'object') {
```

Comment:
same comment as above - is this a better display of object values, and if so should we also use it in `formatValue`?

Reply:
This is made only with summary statistics in mind. This may not work so well for deeply nested objects. In principle, the objects (not supported yet) often destroy the right column anyway and we should think about a different solution. I don't think there's a good way to fit them into the right column and this won't make it much better. So yes, there's work to be done here, but I don't think as part of this PR.

Reply:
Make sense!